go/internal/initutil/ipfs.go

```diff
@@ -37,6 +40,10 @@ func (m *Manager) SetupLocalIPFSFlags(fs *flag.FlagSet) {
 	fs.DurationVar(&m.Node.Protocol.MaxBackoff, "p2p.max-backoff", time.Minute, "maximum p2p backoff duration")
 	fs.BoolVar(&m.Node.Protocol.LocalDiscovery, "p2p.local-discovery", true, "local discovery")
 	fs.Var(&m.Node.Protocol.RdvpMaddrs, "p2p.rdvp", `list of rendezvous point maddr, ":dev:" will add the default devs servers, ":none:" will disable rdvp`)
+	fs.BoolVar(&m.Node.Protocol.Tor.Enabled, "tor.enable", false, "If true tor will be enabled.")
+	fs.StringVar(&m.Node.Protocol.Tor.BinaryPath, "tor.binary-path", "", "If set berty will use this external tor binary instead of his builtin one.")
+	fs.BoolVar(&m.Node.Protocol.Tor.DontSetListen, "tor.dont-set-listen", false, "If true tor will not auto set his listen address, allowing you to replace by an other one.")
```

Comment:
I don't get how you want to use this

Why not just having something like:


```suggestion
	fs.StringVar(&m.Node.Protocol.Tor.Listen, "tor.listen", tor.NopMaddr3Str, "comment")
```

Reply:
By default using `tor.enable` add the `tor.NopMAddr3Str` (`/onion/aaaaa...:1`) to the listen list.
Setting `DontSetListen` don't do that.

That mainly for debug purpose (or using onion 2), but I got a better idea, I'll remove `tor.dont-set-listen` and scan the listen list and if a `/onion{,3}/...` maddr is found the default tor listen address will not be added.

Reply:
yes but, your solution:
1. can only unset, but cannot set a custom value from the CLI
2. makes the `-h` not showing what is the default value